### PR TITLE
[Bots] ATO detections + Detection Tags/IDs in Analytics

### PR DESCRIPTION
**Pull request by patriciasantaana**
### Summary

ATO detections 

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/bots/concepts/detection-ids.mdx [hunk 1]
```diff
@@ -73,3 +84,37 @@ and not any(cf.bot_management.detection_ids[*] in {3355446 12577893})
...
+5. Select **Save as draft** to return to it later, or **Deploy** to deploy the rule.
+
+```js title="Rule example"
+
+(any(cf.bot_management.detection_ids[*] eq 201326593))
+```
+
+### Limit logins with account takeover detections
+
+Rate limiting rules can limit the number of logins from a particular IP, JA4 Fingerprint, or country.
```
Comment (Oxyjun) [inline src/content/docs/bots/concepts/detection-ids.mdx:107]:
```suggestion
Rate-limiting rules can limit the number of logins from a particular IP, JA4 Fingerprint, or country.
```

File: src/content/docs/bots/concepts/detection-ids.mdx [hunk 2]
```diff
@@ -73,3 +84,37 @@ and not any(cf.bot_management.detection_ids[*] in {3355446 12577893})
...
+
+To use rate limiting rules with account takeover detections:
```
Comment (Oxyjun) [inline src/content/docs/bots/concepts/detection-ids.mdx:109]:
```suggestion
To use rate-limiting rules with account takeover detections:
```

Comment (Oxyjun) [reply src/content/docs/bots/concepts/detection-ids.mdx:109]: So I think this should be `rate-limiting rule`, but I can see that maybe the UI components aren't following that convention - pls feel free to ignore:D

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/bots/concepts/detection-ids.mdx [hunk 3]
```diff
@@ -73,3 +84,37 @@ and not any(cf.bot_management.detection_ids[*] in {3355446 12577893})
...
+```js title="Rule example"
+
+(any(cf.bot_management.detection_ids[*] eq 201326593))
+```
+
+### Limit logins with account takeover detections
+
+Rate-limiting rules can limit the number of logins from a particular IP, JA4 Fingerprint, or country.
+
+To use rate-limiting rules with account takeover detections:
```
Comment (patriciasantaana) [inline src/content/docs/bots/concepts/detection-ids.mdx]:
```suggestion
Rate limiting rules can limit the number of logins from a particular IP, JA4 Fingerprint, or country.

To use rate limiting rules with account takeover detections:
```

**Pull request merged**